tests/network/vmi_subdomain.go

```diff
@@ -88,10 +95,12 @@ var _ = SIGDescribe("Subdomain", func() {
 
 			Expect(assertFQDNinGuest(vmi, expectedFQDN)).To(Succeed(), "failed to get expected FQDN")
 		},
-			Entry("with Masquerade binding and subdomain", fedoraMasqueradeVMI, subdomain),
-			Entry("with Bridge binding and subdomain", fedoraBridgeBindingVMI, subdomain),
-			Entry("with Masquerade binding without subdomain", fedoraMasqueradeVMI, ""),
-			Entry("with Bridge binding without subdomain", fedoraBridgeBindingVMI, ""),
+			Entry("with Masquerade binding and subdomain", fedoraMasqueradeVMI, subdomain, ""),
+			Entry("with Masquerade binding and subdomain and hostname", fedoraMasqueradeVMI, subdomain, hostname),
+			Entry("with Bridge binding and subdomain", fedoraBridgeBindingVMI, subdomain, ""),
+			Entry("with Bridge binding and subdomain and hostname", fedoraBridgeBindingVMI, subdomain, hostname),
+			Entry("with Masquerade binding without subdomain", fedoraMasqueradeVMI, "", ""),
+			Entry("with Bridge binding without subdomain", fedoraBridgeBindingVMI, "", ""),
```

Comment:
I do not really understand why we care about `hostname` without `subdomain` in the `subdomain` tests.

If this is not a common usage for users, I do not even think we should cover it.
I saw there are other tests that are using `Spec.Hostname` (e.g. migration, services), I guess they are covering some aspect of it.

The addition of e2e is expensive, we need a very good reason to add more.

Reply:
Good point, but those tests wasn't added in this PR, and it's not the scope of this PR.
Should I raise new PR or adding new commit here if all agree it's irrelevant?

Reply:
For now, I suggest you add the `domain` thing in the existing tests, if this is the expected common user configuration. I want to avoid adding more e2e tests.

I hope @AlonaKaplan can comment on removing these no-subdomain scenarios.

Reply:
I mean that on the existing tests, you add an explicit domain value (`domain := "testhostname"`) instead of adding more tests.